src/aws.yml

```diff
@@ -12,6 +12,18 @@
     # The instance types used for almost all the instances expose EBS
     # volumes as NVMe block devices, so that's why we need nvme here.
     - nvme
+    # This role forces the installation of python2, which breaks any
+    # pip modules that are installed afterwards because Ansible will
+    # see that /usr/bin/python exists and use it.  In particular,
+    # Ansible will try to install python-apt during idempotence; since
+    # this package does not exist in Debian 11 (Bullseye) that causes
+    # Ansible to fail.
+    #
+    # I have created aws/efs-utils#57 to encourage the good folks at
+    # aws/efs-utils to support python3 in their packaging.  (The code
+    # itself already supports python3.)  Until they change that, this
+    # role is the last thing that Ansible installs.
```

Comment:
Is this still true? Per [v1.29.1](https://github.com/aws/efs-utils/releases/tag/v1.29.1) they updated the Python dependency to Python 3.

Reply:
Please see cisagov/skeleton-packer#88.